akka-operator/templates/035-commercial-credentials-secret.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if ne .Values.lightbendSubscription.username "false" }}
+## This is based on the Helm How-To trick to create image pull secrets (https://helm.sh/docs/howto/charts_tips_and_tricks/#creating-image-pull-secrets)
+apiVersion: v1
+kind: Secret
+metadata:
+  name: commercial-credentials
+  labels:
+    app.kubernetes.io/component: credentials
+type: "kubernetes.io/dockerconfigjson"
+data:
+  .dockerconfigjson: {{ template "imagePullSubscriptionSecret" . }}
```

Comment:
Embedding JSON that's created on the fly is done via a template on the `_helpers.tpl` below. I tried doing it in a single pass, it didn't go well. 

The trick on the HOW-TO at https://helm.sh/docs/howto/charts_tips_and_tricks/#creating-image-pull-secrets recommends using the YAML + the .tpl bit and that's how I did it.